pkg/manager/getconfig.go

```diff
@@ -28,6 +29,7 @@ import (
 func (m *Manager) GetTargetConfig(target string, configname string, path string, layer int) ([]*change.ConfigValue, error) {
```

Comment:
I wonder if we should have a separate method GetOpState to handle the Op State stuff